redaxo/src/addons/debug/boot.php

```diff
@@ -35,7 +35,7 @@
         }
 
         foreach ($timings['timings'] as $i => $timing) {
-            if ($timing['end'] - $timing['start'] > 0.001) {
+            if ($timing['end'] - $timing['start'] >= 0.001) {
```

Comment:
Sind stellen die 0.001 als ausführungszeit haben wirklich interessant?

Hatte ich bewusst „ausgeblendet“ damit es übersichtlicher wird

Reply:
Die Zeile ist von mir, den Part hatte ich in Marcels initialem PR für Clockwork ergänzt.
In der Realität wird es ja eher nicht wirklich einen Unterschied machen, ob wir nun sagen "größer 1ms" oder "ab 1ms" (größer gleich). Aber mir gefiel jetzt doch besser, alles ab 1ms anzuzeigen.

Reply:
Also aktuell zeigen wir ab 1001μs an, neu dann ab 1000μs